### DISCUSSION TITLE
Aligning multiple generated bins on a grid

### DISCUSSION THREAD

**Discussion by delneet** [Le0Michine/FusionGridfinityGenerator#116] [Q&A]
What would be the advised way to align multiple generated bins on a grid within fusion?
Right now I'm targeting a corner edge, but that doesn't account for the clearances.
Or is there a way to specify the start x/y(z) positions? Or maybe another generated bin as starting point?

**Comment by grantlesueur** [comment]
@delneet I needed to align my first grid to the corner of a drawer I modelled - I addressed this issue by opening the grid model.   I then  created a sketch on the side of the grid and projected / placed a sketch point on the extreme vertex of that side of the Grid.

With the sketch visible I was able to then create a rigid joint for the Grid to the corner of my drawer, using the sketch point as a snap point for the joint.

After that I use rigid joints to connect each bin to each grid position I wanted.  

The best approach I came up for for this is to apply the joint on the mid point (of the plane) of the bin base surface to the same midpoint of the intended mating surface of the gird.   Here you need to be careful joining vertical face with the mating vertical face and not a chamfer face as this will cause your bin to stick out 45 degrees (Ask me how I know).  Zooming in real tight is the key here.

Once you get the hang of it, it is rather fun.   Good luck

**Reply by delneet** [reply]
Wow! I'm going to have to read this 3 times to understand what you just said 😅 
<sub>disclaimer: I'm a real novice</sub>
Good thing you found a way though. I'm going to try and figure it out!
